### PR TITLE
172 datetime

### LINKED ISSUE #172: Support for DateTime element timezones
### Motivation

As part of [#356](https://github.com/kontent-ai/delivery-sdk-net/pull/357) we added new `DateTimeContentConverter` to Delivery .NET SDK to have a possibility to convert DateTimeElement to custom `DateTimeContent` model, which has the following interface:

```
public interface IDateTimeContent
{
    public DateTime? Value { get; }
    public string DisplayTimezone { get; }
}
```
At the same time, we kept the possibility to convert DateTimeElement to system `DateTime?` type the same as it worked before.

It's needed to extend the model generator's functionality to let customers choose, whether they want to convert DateTimeElement to simple `DateTime?` value (previous behavior) or to custom `IDateTimeContent` model.

### Proposed solution

It seems reasonable to use similar approach as for RichTextElement converting. Model generator creates a property of `string` or custom `IRichTextContent` model depending on `--structuredmodel <True|False>` parameter value.
As we discussed on grooming, we can change type of `structuredmodel` property from current `bool` to `string`, so that it will contain a list of element types, for which custom type will be generated. Currently, there will be two possible elements, for with structured model could be generated:

- RichTextElement
- DateTimeElement

So, possible `structuredmodel` property format could be
`--structuredmodel richtext|datetime`

### Additional context

Add any other context, screenshots, or reference links about the feature request here.

### PR DESCRIPTION
**Pull request by Sevitas**
### Motivation

Which issue does this fix? Fixes #172 

### Checklist

- [x] Code follows coding conventions held in this repo
- [x] Automated tests have been added
- [x] Tests are passing
- [x] Docs have been updated (if applicable)
- [x] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

Generate structured elements for RT, DT. Generate elements without structured option.

### CODE REVIEW

**Review by Simply007 — CHANGES_REQUESTED** [review]
Let's try to avoid the breaking change. Leave the "True" value for the structured model as a possible solution, but **deprecated**.

* Once the user upgrade and uses `--structuredMode True` add logging that the value is deprecated
* Note that to readme as well (and ideally mark it by `System.Obsolete`)

**Comment by Sevitas** [comment]
> Let's try to avoid the breaking change. Leave the "True" value for the structured model as a possible solution, but **deprecated**.
> 
> * Once the user upgrade and uses `--structuredMode True` add logging that the value is deprecated
> * Note that to readme as well (and ideally mark it by `System.Obsolete`)

Should be fixed.

File: README.md [hunk 1]
```diff
@@ -27,40 +27,40 @@ The recommended way of obtaining this tool is installing it as a [.NET Tool](htt
 #### Global Tool
 
 - `dotnet tool install -g Kontent.Ai.ModelGenerator`
-- `KontentModelGenerator --projectid "<projectid>" [--namespace "<custom-namespace>"] [--outputdir "<output-directory>"] [--withtypeprovider <True|False>] [--structuredmodel <True|False>] [--filenamesuffix "<suffix>"]`
+- `KontentModelGenerator --projectid "<projectid>" [--namespace "<custom-namespace>"] [--outputdir "<output-directory>"] [--withtypeprovider <True|False>] [--structuredmodel <"{structured_model}"] [--filenamesuffix "<suffix>"]`
```
Comment (winklertomas) [inline README.md:30]: unnecessary quotation marks? You didn't use them in https://github.com/kontent-ai/delivery-sdk-net/pull/361/files

And missing '>'. The same applies to examples bellow

**Review by winklertomas — CHANGES_REQUESTED** [review]

**Review by winklertomas — APPROVED** [review]
Review OK

**Pull request merged**